### PR TITLE
Adds docs for custom roles in Calico Cloud

### PR DESCRIPTION
**Pull request by ctauchen**

### CODE REVIEW

File: calico-cloud_versioned_docs/version-3.15/users/creating-and-assigning-custom-roles.mdx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+import IconUser from '/img/icons/user-icon.svg';
+
+# Creating and assigning custom roles
+
+As an administrator, you can create custom, cluster-specific roles that restrict users to particular functions on a cluster.
+
+## Overview
+
+Calico Cloud comes with a set of predefinied global roles that let you give permissions to users based on what they need to do.
```
Comment (bmckercher123) [inline calico-cloud_versioned_docs/version-3.15/users/creating-and-assigning-custom-roles.mdx]: {{prodname}} (check across the doc)

File: calico-cloud_versioned_docs/version-3.15/users/creating-and-assigning-custom-roles.mdx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+For example, a user with the Security global role has a broader set of permissions than a user with the Viewer global role.
+These permissions apply to all clusters.
+
+But in some cases these global roles can be too broad.
+
+By creating and assigning custom roles, administrators can be much more discriminating about what permissions they give users.
```
Comment (bmckercher123) [inline calico-cloud_versioned_docs/version-3.15/users/creating-and-assigning-custom-roles.mdx]: Doc point of view is uneven. Doc goes back and forth between "you" and "administrators" and "to your users". First sentence introduces "administrator" as the actor, so I would say "you" and "users".

File: calico-cloud_versioned_docs/version-3.15/users/creating-and-assigning-custom-roles.mdx [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+For example, you could create a role that gives a user the ability to modify network policy for a particular tier and namespace and gives view access to all other network policies.
```
Comment (bmckercher123) [inline calico-cloud_versioned_docs/version-3.15/users/creating-and-assigning-custom-roles.mdx]:
```suggestion
For example, you could create a role that allows the user to modify network policy for a particular tier and namespace, and view access to all other network policies.
```

File: calico-cloud_versioned_docs/version-3.15/users/creating-and-assigning-custom-roles.mdx [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+Permissions are assigned on a cluster-by-cluster basis.
+
+## Required permissions for common Calico Cloud features
+
+Certain permissions are required for a user to access common Calico Cloud features.
+
```
Comment (bmckercher123) [inline calico-cloud_versioned_docs/version-3.15/users/creating-and-assigning-custom-roles.mdx:26]: The inconsistent wrapping in the table makes it hard to grok some cells. I'm not fond of it myself, but using a dash as a bullet may help with readability. 
- View or Modify Alerts
and
- View Event Logs or View All Logs

File: calico-cloud_versioned_docs/version-3.15/users/creating-and-assigning-custom-roles.mdx [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+## Create a custom role and attach permissions
+
+1. Click the user icon <IconUser width="20"/> > **Manage Team**.
+1. Under the **Roles** tab, click **Add Role**, enter a name and description for the custom role, and then click **Save**.
+1. Select the cluster you want to want the role to apply to by clicking **Cluster:** and choosing the cluster.
+1. Locate your new role in the list, select **Action** > **Manage permissions** > **Edit**, and the click **Add Permission**.
+1. Under **Permission**, choose a permission type from the list.
+   Depending on the permission, you may also need to choose a namespace or policy tier.
+1. (optional) Click **Add permission** to add more permissions to your role for this cluster.
+1. Click **Save** to attach these permissions to the role for this cluster.
```
Comment (bmckercher123) [inline calico-cloud_versioned_docs/version-3.15/users/creating-and-assigning-custom-roles.mdx]: No need to introduce "attach"...

1. Click **Save** to save these permissions to the role for this cluster. 
1. (optional) If you want to add the permissions to another cluster, repeat steps 3 to 7 for the cluster.

File: calico-cloud_versioned_docs/version-3.15/users/creating-and-assigning-custom-roles.mdx [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+
+Certain permissions are required for a user to access common Calico Cloud features.
+
+| Feature area | Required permissions | Notes |
+| --| -- | -- |
+| Alerts | **View** or **Modify Alerts**<br/>and<br/>**View Event Logs** or **View All Logs** | |
+| Anomaly detection | **View** or **Modify Alerts** and **View Event Logs** or **View All Logs**| |
+| Compliance reports | **View Compliance Reports** | |
+| Dashboard | **View All Logs**<br/>and<br/>**View Global Network Sets** or **View Network Sets**<br/>and<br/>(optional) **View Compliance Reports** | These permissions are required for the dashboard to fully populate. Limited metrics are enabled just by having access to a cluster.<br/> |
+| Network policies | **View** or **Modify Policies**<br/>or<br/>**View** or **Modify Global Policies**<br/>and<br/>(optional) **View Audit Logs** or **View All Logs** | The **Policies** permissions are scoped to one or more namespaces. The **Global Policies** permissions apply to the whole cluster. These permissions are also scoped by [policy tier](../network-policy/index.mdx). |
```
Comment (bmckercher123) [inline calico-cloud_versioned_docs/version-3.15/users/creating-and-assigning-custom-roles.mdx]: policy tier should not go to the index. Better straight to the doc on tiers: /network-policy/policy-tiers/

Also, the paragraph unevenly uses "apply" and "permissions are scoped". Apply is good.

Comment (bmckercher123) [reply calico-cloud_versioned_docs/version-3.15/users/creating-and-assigning-custom-roles.mdx]: I had to read this twice so suggest:

These permissions are required for the dashboard to fully populate. All users are granted limited dashboard metrics by having access to a cluster.

**Review by bmckercher123 — DISMISSED** [review]
Minor nits.

File: calico-cloud_versioned_docs/version-3.15/users/creating-and-assigning-custom-roles.mdx (hunk 1 above)
Comment (ctauchen) [reply calico-cloud_versioned_docs/version-3.15/users/creating-and-assigning-custom-roles.mdx]: done

File: calico-cloud_versioned_docs/version-3.15/users/creating-and-assigning-custom-roles.mdx (hunk 2 above)
Comment (ctauchen) [reply calico-cloud_versioned_docs/version-3.15/users/creating-and-assigning-custom-roles.mdx]: done

File: calico-cloud_versioned_docs/version-3.15/users/creating-and-assigning-custom-roles.mdx (hunk 3 above)
Comment (ctauchen) [reply calico-cloud_versioned_docs/version-3.15/users/creating-and-assigning-custom-roles.mdx]: done, as a hybrid

File: calico-cloud_versioned_docs/version-3.15/users/creating-and-assigning-custom-roles.mdx (hunk 6 above)
Comment (ctauchen) [reply calico-cloud_versioned_docs/version-3.15/users/creating-and-assigning-custom-roles.mdx]: done, done, and done

File: calico-cloud_versioned_docs/version-3.15/users/creating-and-assigning-custom-roles.mdx (hunk 5 above)
Comment (ctauchen) [reply calico-cloud_versioned_docs/version-3.15/users/creating-and-assigning-custom-roles.mdx]: Yes! Thanks!

**Review by gcosgrave — APPROVED** [review]
LGTM

**Pull request merged**